conda-osx-64.lock

```diff
@@ -1,5 +1,5 @@
 # platform: osx-64
```

Comment:
not sure if there are any new deps in here

Reply:
Not as far as I see, but some of the deps got reordered which makes it a bit hard to tell.